packages/plugins/plugin-navtree/src/components/NavTree/NavTree.stories.tsx

```diff
@@ -36,13 +38,54 @@ faker.seed(1234);
 
 const StoryState = defineCapability<{ tab: string }>('story-state');
 
+// TODO(burdon): How to adjust existing surfaces for this test?
+// TODO(burdon): How to toggle attention in this test?
```

Comment:
I’ve been struggling with this too, need to investigate or build scaffolding for attention in storybook.

Reply:
@wittjosiah

Reply:
@thure there's an attention decorator for stories already, I can help you set that up if you're not sure where to get started